packages/flutter_tools/test/integration/flutter_test_driver.dart

```diff
@@ -41,40 +54,58 @@ class FlutterTestDriver {
     _stderr.stream.listen(_errorBuffer.writeln);
 
     // This is just debug printing to aid running/debugging tests locally.
-    if (_printJsonAndStderr) {
-      _stdout.stream.listen(print);
-      _stderr.stream.listen(print);
-    }
+    _stdout.stream.listen(debugPrint);
+    _stderr.stream.listen(debugPrint);
 
     // Set this up now, but we don't wait it yet. We want to make sure we don't
     // miss it while waiting for debugPort below.
     final Future<Map<String, dynamic>> started = _waitFor(event: 'app.started');
 
     if (withDebugger) {
       final Future<Map<String, dynamic>> debugPort = _waitFor(event: 'app.debugPort');
-      final String wsUri = (await debugPort)['params']['wsUri'];
-      vmService = new VMServiceClient.connect(wsUri);
+      final String wsUriString = (await debugPort)['params']['wsUri'];
+      final Uri uri = Uri.parse(wsUriString);
+      // Proxy the steram/sink for the VM Client so we can debugPrint it.
```

Comment:
steram => stream

Reply:
I think I was subconsciously adding typos for you! 😁 (both fixed)